spec/services/nginx_spec.rb

```diff
@@ -3,11 +3,27 @@
 require 'spec_helper'
 set :os, family: 'redhat', release: '9', arch: 'x86_64'
 
-describe package('nginx') do
-  it { should be_installed }
-end
+describe 'Checking nginx' do
+  describe package('nginx') do
+    it { should be_installed }
+  end
+
+  describe service('nginx') do
+    it { should be_enabled }
+    it { should be_running }
+  end
```

Comment:
This needs first to be checked in chef, for knowing if the node has nginx or not, in a custom node without nginx, test will fail